packages/hash/frontend/src/components/BlockLoader/shared.ts

```diff
@@ -0,0 +1,112 @@
+import { JsonObject } from "@blockprotocol/core";
+import { BlockGraph, Entity } from "@blockprotocol/graph";
+import get from "lodash/get";
+import set from "lodash/set";
```

Comment:
```suggestion
import { get, set } from "lodash";
```

I think in most cases in HASH we're using named imports rather than specific modules.

Reply:
https://github.com/hashintel/hash/pull/918/commits/b4eae05c32e1e29eaf0078b90e790874984398da